safe_core/src/client/mod.rs

```diff
@@ -1131,6 +1135,7 @@ where
 }
 
 /// Create a new mock balance at an arbitrary address.
+#[cfg(any(test, feature = "testing"))]
```

Comment:
This should not be feature-gated. We use this to create test coins in the real-network also.
 Eg: https://github.com/maidsafe/safe_client_libs/blob/32531bea3383cf245a7211dd913073e71fbf40e0/safe_authenticator/src/ffi/mod.rs#L60